packages/web-components/fast-foundation/src/tooltip/tooltip.ts

```diff
@@ -254,7 +254,7 @@ export class FASTTooltip extends FASTElement {
      *
      * @internal
      */
-    private set visible(value: boolean | undefined) {
+    private setVisible(value: boolean | undefined) {
```

Comment:
I'm ok with this since it's a private API.

Reply:
I'm curious why this needs to change though - this is a private setter...what's the relationship to this PR?

Reply:
The suggestion [here](https://api-extractor.com/pages/messages/ae-setter-with-docs/) from API Extractor is to remove the comment, which seems reasonable to me. I'd prefer that we not change API's as a practice to conform to tooling.

Reply:
If we remove the comment from the setter it will remove the line from the README file. Are we alright with that? Seems preferable to have the field listed even if it's readonly.

Agreed that API Extractor should handle this better, but it's a private function only called in the one place in this class, so this doesn't represent a change to the API.

Reply:
I guess this is fine, but definitely not ideal. I can understand their reasoning for getters and setters being a single consideration.